hooks/hookexecution/execution.go

```diff
@@ -166,8 +170,13 @@ func handleHookResponse[P any](
 	ctx executionContext,
 	payload P,
 	hr hookResponse[P],
+	metricEngine metrics.MetricsEngine,
 ) (P, HookOutcome, *RejectError) {
 	var rejectErr *RejectError
+	labels := metrics.ModuleLabels{Module: hr.HookID.ModuleCode, Stage: ctx.stage, PubID: ctx.accountId}
+	metricEngine.RecordModuleCalled(labels)
+	metricEngine.RecordModuleDuration(labels, hr.ExecutionTime)
```

Comment:
I recommend combining `RecordModuleCalled` and `RecordModuleDuration` since they're only called here once as a pair and it makes sense IMHO for a call to be directly associated with a duration.

Reply:
makes sense, updated